src/__tests__/reducers/articles.test.js

```diff
@@ -0,0 +1,140 @@
+import articlesReducer from '../../reducers/articlesReducer';
```

Comment:
@djallas, it's a good practice to have the test file having the same name as the file being tested, so this file can be renamed to something like: `src/__tests__/reducers/articlesReducers.test.js`

Reply:
Thanks, well noted.